opengever/maintenance/scripts/fix_broken_intid_references.py

```diff
@@ -13,20 +13,21 @@
 The --fix will then read these paths and fix each object. It will commit the transaction
 every few items and will update the the json with the remeaing items.
 """
-from five.intid.intid import moveIntIdSubscriber
-from ftw.upgrade.progresslogger import ProgressLogger
-from opengever.maintenance.debughelpers import setup_app
-from opengever.maintenance.debughelpers import setup_option_parser
-from opengever.maintenance.debughelpers import setup_plone
-from plone import api
-from zope.component import getUtility
-from zope.intid import IIntIds
 import json
 import logging
 import os
 import sys
 import time
+
 import transaction
+from five.intid.intid import moveIntIdSubscriber
+from ftw.upgrade.progresslogger import ProgressLogger
+from plone import api
+from zope.component import getUtility
+from zope.intid import IIntIds
+
+from opengever.maintenance.debughelpers import (setup_app, setup_option_parser,
+                                                setup_plone)
```

Comment:
Your isort seems to be missconfigured. Please keep the previous import order styling (no grouping and single line imports)